test/es-module/test-esm-loader-not-found.mjs

```diff
@@ -0,0 +1,25 @@
+import '../common/index.mjs';
+import { path } from '../common/fixtures.mjs';
+import { ok } from 'assert';
+import { spawn } from 'child_process';
+import { execPath } from 'process';
+
+const child = spawn(execPath, [
+  '--experimental-loader',
```

Comment:
Since `--loader` also exists and aliases to `--experimental-loader`, should we use that instead to avoid having to change them later? (I'm mentioning now because these tests will be the "do this" example for ESM tests)

Reply:
I considered this and I’m uncertain what’s best. There are many other tests that mention `--experimental-loader`, so I thought it was probably better to be consistent, and we would update all the mentions of the flag to be non-experimental at once.

Reply:
Ahh, good point. Withdrawn since the "problem" already exists.